pkg/kubelet/remote/remote_runtime.go

```diff
@@ -424,6 +424,20 @@ func (r *RemoteRuntimeService) UpdateRuntimeConfig(runtimeConfig *runtimeapi.Run
 	return nil
 }
 
+func (r *RemoteRuntimeService) GetRuntimeConfigInfo() (*runtimeapi.ActiveRuntimeConfig, error) {
+	ctx, cancel := getContextWithTimeout(r.timeout)
+	defer cancel()
+
+	resp, err := r.runtimeClient.GetRuntimeConfigInfo(ctx, &runtimeapi.GetRuntimeConfigInfoRequest{})
+	if err != nil {
+		errorMessage := "Failed to get runtime configuration details"
```

Comment:
nit: error message should start lower always.  i know we have a similar problem in same file with `RunPodSandbox`.